include-what-you-use.1

```diff
@@ -72,10 +72,32 @@ Maximum line length for includes.
 Note that this only affects the comments and their alignment, the maximum line
 length can still be exceeded with long filenames (default: 80).
 .TP
+.BI \-\-comment_style= verbosity
+Controls the style and verbosity of \(lqwhy\(rq comments at the end of
+suggested includes.
+.IR verbosity
+options are:
```

Comment:
```suggestion
suggested includes. Options for
.I verbosity
are:
```

To make sure we start the sentence with a capital letter. Also, to quote [Knuth](https://jmlr.csail.mit.edu/reviewing-papers/knuth_mathematical_writing.pdf): “Don’t start a sentence with a symbol.”

Also `.I` means italic, `.IR` means alternating italic and regular (“roman”). We have just one word here.

Reply:
Committed, thank you :)